apps/web/src/components/mobile-nav/dark-toggle.tsx

```diff
@@ -0,0 +1,33 @@
+"use client";
+
+import * as React from "react";
+import { Button, RenderIcon } from "@tietokilta/ui";
+import { useTheme } from "next-themes";
+
+export function DarkModeToggle() {
+  const { theme, setTheme } = useTheme();
+  const [mounted, setMounted] = React.useState(false);
+
+  React.useEffect(() => {
+    setMounted(true);
+  }, []);
+
+  if (!mounted) {
+    return <div />;
+  }
+
+  return (
+    <Button
+      variant="ghost"
+      onClick={() => {
+        setTheme(theme === "dark" ? "light" : "dark");
```

Comment:
Instead of a pure toggle, what if this was a menu with "System" / "Light" / "Dark" options?